plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * WooCommerce Admin Helper - React admin interface
+ *
+ * @package WooCommerce\Admin\Helper
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC_Helper Class
+ *
+ * The main entry-point for all things related to the Helper.
+ */
+class WC_Helper_Admin {
+
+	public static function load() {
+		add_filter( 'woocommerce_admin_shared_settings', array( __CLASS__, 'add_marketplace_settings' ) );
+	}
+	public static function add_marketplace_settings( $settings ) {
+		$settings['wccomHelper'] = [
+			'isConnected' => WC_Helper::is_site_connected(),
+			'connectURL'  => self::getConnectionURL(),
```

Comment:
We will need to return to review this URL still works later once this issue is resolved: https://github.com/orgs/Automattic/projects/366/views/43?pane=issue&itemId=32504577